doc/articles/get-started-rider.md

```diff
@@ -8,7 +8,7 @@
 
 |                       | Windows (UWP/WinUI) | Android | iOS  | Wasm | macOS | Skia-GTK (Linux) | Skia-WPF |
 | --------------------- | ------------------- | ------- | ---- | ---- | ----- | -------- | -------- |
-| **Rider for Linux**   | ❌                   | ✔️       | ❌    | ✔️†   | ❌     | ✔️        | ❌        |
+| **Rider for Linux**   | ❌                   | ✔️†      | ❌    | ✔️†   | ❌     | ✔️        | ❌        |
```

Comment:
Thanks for taking the time to edit the documentation! As it's not supported, let's make it a cross instead.
```suggestion
| **Rider for Linux**   | ❌                   | ❌†      | ❌    | ✔️†   | ❌     | ✔️        | ❌        |
```